scripts/protocgen.sh

```diff
@@ -43,11 +43,11 @@ Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
   -I "proto" \
   -I "third_party/proto" \
   --go-cosmos2_out=\
-Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,\
-Mgoogle/protobuf/struct.proto=github.com/gogo/protobuf/types,\
-Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,\
-Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types,\
-Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
+google/protobuf/duration.proto=github.com/gogo/protobuf/types,\
+google/protobuf/struct.proto=github.com/gogo/protobuf/types,\
+google/protobuf/timestamp.proto=github.com/gogo/protobuf/types,\
+google/protobuf/wrappers.proto=github.com/gogo/protobuf/types,\
+google/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
```

Comment:
Are these `M`s unneeded?

Reply:
This was causing some issues when using both `google.protobuf.Duration` and `google.protobuf.Timestamp` in the same package: `protoc-gen-go-cosmos2: Go package "github.com/gogo/protobuf/types" has inconsistent names duration (google/protobuf/duration.proto) and timestamp (google/protobuf/timestamp.proto)`, coming from https://github.com/protocolbuffers/protobuf-go/blob/master/compiler/protogen/protogen.go#L375